CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## TBD
 * Updated `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
```

Comment:
Not sure why this doesn't have a heading already, but this should be under a `### Changed`